docs/src/main/asciidoc/Getting_Started_Developing_Applications_Guide.adoc

```diff
@@ -60,7 +60,7 @@ JBoss AS 7, but with Ruby.
 
 :leveloffset: +1
 
-include::_extras/Developing_JSF_Project_Using,_Maven_and_IntelliJ.adoc[]
+include::_extras/Developing_Jakarta_Server_Faces_Project_Using,_Maven_and_IntelliJ.adoc[]
```

Comment:
Just a small comment, since we're renaming this file, it would be good to remove the comma after `Using,` since it looks like that must have been a typo initially.

Reply:
Thanks @fjuma ! Done the changes :)